opsdroid/connector/facebook/__init__.py

```diff
@@ -0,0 +1,99 @@
+"""A connector for Facebook Messenger."""
+import json
+import logging
+
+import aiohttp
+
+from opsdroid.connector import Connector
+from opsdroid.message import Message
+
+
+_LOGGER = logging.getLogger(__name__)
+_FACEBOOK_SEND_URL = "https://graph.facebook.com/v2.6/me/messages" \
+                     "?access_token={}"
+
+
+class ConnectorFacebook(Connector):
+    """A connector for Facebook Messenger."""
```

Comment:
We are currently trying to implement google style docstrings through the whole code base, could you replace these docstrings into a google style ones?